salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -84,4 +87,7 @@ Configure registry IP in containerd conf:
     - makedirs: true
     - contents: |
         [plugins.cri.registry.mirrors."{{ repo.registry_endpoint }}"]
-          endpoint = ["http://{{ registry_ip }}:{{ registry_port }}"]
+        endpoint = ["http://{{ registry_ip }}:{{ registry_port }}"]
+
+        [debug]
+        level = "{{ 'debug' if metalk8s.debug else 'info' }}"
```

Comment:
What's the purpose of using both the containerd option `--log-level` and this ? I think we should stick to only one way of doing this, and putting the option in this file seems the better option to me.

Reply:
So:
- `--log-level` seems to adjust the main daemon's logging, which ends up in journald
- the `[debug]` config section configures the debug socket, at `/run/containerd/debug.sock` by default (I didn't try listening on this socket so I don't what gets sent in there :sweat_smile:)

Reply:
Oh ok, I see the difference, I thought it was the same thing, I've checked the man (containerd-config.toml) and effectively it is not related.
Still, that's weird that there is no option to change daemon log level within this configuration file.